pages/textbook-beta/course/machine-learning-course.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+  <main class="quantum-machine-learning-course-page">
+    <TextbookBetaContentMenuSection
+      class="quantum-machine-learning-course-page__content-menu"
+    />
+    <AppPageHeaderWithImage :cta="startLearningCTA" :back-link="backToTextbookHomeLink">
+      <template slot="title">
+        {{ headerTitle }}
+      </template>
+      <template slot="description">
+        <p v-text="headerDescription" />
+      </template>
+      <template slot="image">
+        <img class="app-page-header-with-img__image" :src="headerImg">
+      </template>
+    </AppPageHeaderWithImage>
+    <PrerequisiteMaterialSection class="quantum-machine-learning-course-page__section" :data="qmlCoursePrerequisites" />
+    <CoursePagesSection class="quantum-machine-learning-course-page__section" :courses="courses" :img-base="imageUrlBase" />
+  </main>
+</template>
+
+<script lang="ts">
+import { Course, Prerequisite } from 'constants/textbookBetaContent'
+import { Component } from 'vue-property-decorator'
+import QiskitPage from '~/components/logic/QiskitPage.vue'
+import { GeneralLink, textbookBetaQMLStartLearningUrl } from '~/constants/appLinks'
+
+@Component({
+  layout: 'default-max',
+  head () {
+    return {
+      title: 'Quantum machine learning course'
+    }
+  }
+})
+export default class QuantumMachineLearningCoursePage extends QiskitPage {
+  routeName: string = 'quantum-machine-learning-course'
```

Comment:
@frankharkins - regarding this update:

> we decided to go with machine-learning over quantum-machine-learning, so I've updated the overview page to reflect that.

I think this should be updated to reflect that.

Also, should everything else remain with `Quantum machine learning`, `qml`, `class="quantum-machine-learning"`, etc?